src/lib/browse/Filters.svelte

```diff
@@ -141,7 +162,11 @@
   <InterventionColorSelector />
 </CollapsibleCard>
 
-<p>
-  Showing {schemesToBeShown.size} schemes ({counts.route} routes, {counts.area} areas,
-  {counts.crossing} crossings, {counts.other} other)
-</p>
+<CheckboxGroup small>
+  <Checkbox id="show-schemes" bind:checked={show}>
+    Showing {schemesToBeShown.size.toLocaleString()} schemes ({counts.route.toLocaleString()}
```

Comment:
Also adding commas every 3 orders of magnitude here, for consistency with other places in ATIP